examples/pytorch/graphsage_base/README.md

```diff
@@ -0,0 +1,31 @@
+Inductive Representation Learning on Large Graphs (GraphSAGE)
+============
+
+- Paper link: [http://papers.nips.cc/paper/6703-inductive-representation-learning-on-large-graphs.pdf](http://papers.nips.cc/paper/6703-inductive-representation-learning-on-large-graphs.pdf)
+- Author's code repo: [https://github.com/williamleif/graphsage-simple](https://github.com/williamleif/graphsage-simple). Note that the original code is 
+simple reference implementation of GraphSAGE.
+
+Advanced usages, including how to run pure GPU sampling, how to train with PyTorch Lightning, etc., are in the `graphsage/advanced` directory.
```

Comment:
You might want to have something like `[GraphSAGE advanced example](../graphsage/advanced)`

Reply:
Thanks for the suggestion. I updated it with a link.